src/collective/cover/tests/cover.robot

```diff
@@ -44,7 +44,8 @@ Setup Cover Test Case
 Click Add Cover
     Open Add New Menu
     Click Link  css=a#collective-cover-content
-    Page Should Contain  Add Cover
+    Wait Until Page Contains  Add Cover
+    Sleep  1s  Wait for js
```

Comment:
The problem here is that the robot can't click in [Save](https://github.com/collective/collective.cover/blob/master/src/collective/cover/tests/cover.robot#L56) while the js of plone4.csrffixes not do your job. 

@hvelarde why not use Sleep? The tests already takes long. Increase 1 second will not make a difference. We have Sleep in the code:

https://github.com/collective/collective.cover/blob/985d0faafdd3b3401c2ba56d5c0c2d8aaac2b48e/src/collective/cover/tests/test_carousel_tile.robot#L118